Assets/Scripts/CoinManager.cs

```diff
@@ -1,6 +1,6 @@
-﻿using UnityEngine;
-using System.Collections;
-
+﻿using UnityEngine;
+using System.Collections;
+

```

Comment:
Why'd we add these using directives back in? I don't think we use them do we do?